design-proposals/virtiofs-support/virtiofs-feature-gates.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+Virtiofs allows sharing Secrets, ConfigMaps, ServiceAccounts and DownwardAPI (a.k.a., ConfigVolumes) in a dynamic way,
+i.e., any change in those objects are reflected in the VirtualMachine (VM) without restarting the VM. Moreover, PVCs may
+also be shared using virtiofs allowing multiple readers and writers in the shared PVCs.
+
+## Motivation
+Currently, the usage of feature gates are mixed with the feature control. The feature gate `ExperimentalVirtiofsSupport`
+controls whether virtiofsd runs as rootful container inside the launcher pod or not. The feature gate ExperimentalVirtiofsSupport
```

Comment:
There are two sentences very similar:
1. The feature gate `ExperimentalVirtiofsSupport`
controls whether virtiofsd runs as rootful container inside the launcher pod or not.
2. The feature gate ExperimentalVirtiofsSupport controls whether virtiofs can be used, and it automatically runs as rootful container inside the launcher pod when needed.

Reply:
Dropped second sentence.